articles/azure-stack/azure-stack-update-1903.md

```diff
@@ -107,24 +107,31 @@ The following are post-installation known issues for this build version.
 
 ### Portal
 
-<!-- 2930820 - IS ASDK --> 
-- In both the administrator and user portals, if you search for "Docker," the item is incorrectly returned. It is not available in Azure Stack. If you try to create it, a blade with an error indication is displayed. 
+- Azure Stack Dashboard Feedback button is not responding/redirecting. The root cause in the fact that a recent change in Azure was not taken into account in Azure Stack. This change consists of the change of the name of the setting used by the Feedback button from **giveFeedback** to **submitAnIdea**. Release 1904 will have this issue corrected.
```

Comment:
Will check with engineering team if this is something we want to expose.

Reply:
Can remove the additional remarks but the issue should be added as it is valid.

Reply:
Confirmed this is an issue, but I've been asked to use different wording, so I will make this change in a separate PR. Thank you very much for bringing this up!

Reply:
Text to be added: 

In the user portal dashboard, when you try to click on the **Feedback** tile, an empty browser tab opens. As a workaround, you can use [Azure Stack User Voice](http://aka.ms/azurestackuservoice) to file a user voice request.

Reply:
Happy with that, for as long as we list the issue, how much we "publish" about it is a different story ;-) - I will fix merge issues after you merge yours as I still want other changes from this PR to be merged if possible :) Thanks for looking into it! Really appreciate it!

Reply:
Cool, thank you! PRs have now been merged into master, please let me know if you see more issues.

Reply:
So can we merge this PR to master now please?